openfisca_core/commons.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+
+class Dummy(object):
+    """A class that does nothing
+
+    Used by function ``empty_clone`` to create an empty instance from an existing object.
+    """
+    pass
+
+
+def empty_clone(original):
```

Comment:
There is a lot of weird cloning business done in openfisca... To clean another day ;)